com.nonpolynomial.intiface_central.metainfo.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html -->
+<!-- Credits to github/nickavem for the app data, Doomsdayrs for edits -->
+<component type="desktop">
+  <id>com.nonpolynomial.intiface_central</id>
```

Comment:
why does this not match any of the URL below?

Reply:
I believe @qdot once told me that it is because the app is technically under "nonpolynomial" and not via the "intiface.com"?

Upstream also refers to itself that way too.

Reply:
ok.

Reply:
HAH. Hi Hubert. Been a long time since our FxOS days. :D 

Anyways, yeah, nonpolynomial is my company, intiface is the brand for our application suite. The company name doesn't really come up often in relation to the software names though.